lib/chef/knife/user_delete.rb

```diff
@@ -23,21 +23,133 @@ class Knife
     class UserDelete < Knife
 
       deps do
+        require_relative "../org"
         require_relative "../user_v1"
       end
 
       banner "knife user delete USER (options)"
 
+      option :no_disassociate_user,
+        long: "--no-disassociate-user",
+        short: "-d",
+        description: "Don't disassociate the user first"
```

Comment:
disassociate from what? This description isn't very helpful

Reply:
I will update it

Reply:
for disassociate user I will update once this is merge https://github.com/chef/knife-opc/pull/36 as it has changes related to it

Reply:
@tas50 can you please confirm if https://github.com/chef/knife-opc/pull/56 is good to add changes in this PR?